Sources/MeiliSearch/Indexes.swift

```diff
@@ -560,7 +568,7 @@ public struct Indexes {
    value. If the request was sucessful or `Error` if a failure occured.
    */
   public func resetSynonyms(
-    _ completion: @escaping (Result<Task, Swift.Error>) -> Void) {
+    _ completion: @escaping (Result<TaskInfo, Swift.Error>) -> Void) {
```

Comment:
All the `Task` to `TaskInfo` changes in the completion parameter are already changed in this pr #316